metrics/expvar/expvar.go

```diff
@@ -75,9 +81,35 @@ type histogram struct {
 	hist *hdrhistogram.WindowedHistogram
 
 	name   string
+	count  metrics.Counter
 	gauges map[int]metrics.Gauge
 }
 
+// Name returns the name of the histogram
+func (h *histogram) Name() string {
+	return h.name
+}
```

Comment:
I couldn't figure out the type assertion and worked around this in the caller. The calling code would be cleaner if the name and current value could be obtained directly for histograms, counters and gauges. I didn't want to change the metrics.Histogram interface without discussing it with you first.